### DISCUSSION TITLE
Adjust time to next game

### DISCUSSION THREAD

**Discussion by cableghost** [vasqued2/ha-teamtracker-card#66] [General]
This is a great integration! Is it possible to change the game within 18 hours to something greater, ie. 168 hours? 

If there isn't a game within the 18 hrs, the card just displays such. At a minimum, maybe continue to show last game? 

![Screenshot_20231028_121240_Home Assistant.jpg](https://github.com/vasqued2/ha-teamtracker-card/assets/5576447/fefbfb69-12c4-4283-a2ec-f55753425a45)

**Comment by vasqued2** [comment]
The card displays whatever game the API returns.   For some sports like football, upcoming games are returned almost a week in advance.  For others that have games more frequently, the API only returns a game a day or two in advance.  It's a limitation of the API as it only returns so many upcoming games in a league before it maxes out.  Hockey has a ton more games in a week than football so it hits the limit much sooner.
